src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+import React from "react"
+import s from "styled-components"
+import { Title, Subtitle, Text, Link } from "../components/shared/Typography"
+import { STEEL_BLUE, GREEN } from "../utils/constants"
+import Renecycle from '../images/Renecycle.jpg'
+import Hunter from '../images/Hunter Lee _ CEO.jpg'
+import Mina from '../images/Mina Liang _ COO.jpg'
+import Henry from '../images/Henry Lei.jpeg'
+import David from '../images/David Chang.jpeg'
+import Ian from '../images/Ian Gordon.jpeg'
+import Folia from '../images/Folia.jpg'
+import Rosanna from '../images/RosannaHeadshot.jpg'
+import Caroline from '../images/CarolineHeadshot.jpeg'
+import Zachary from '../images/ZacharyHeadshot.jpeg'
+import Stephanie from '../images/StephanieHeadshot.jpg'
+import Ryoma from '../images/RyomaHeadshot.jpg'
+import TeamChaeto from '../images/TeamChaeto.jpg'
+import ChrisNick from '../images/Team Chaeto.jpeg'
+import NASADYA from '../images/NASADYA.jpg'
+import Chaitanya from '../images/Chaitanya_Gulati Headshot.png'
+import Praneeth from '../images/NASADYA_Praneeth Nagarjuna_Headshot.jpeg'
+import Devansh from '../images/NASADYA_Devansh-Sethia_Headshot.jpg'
+import Khadeeja from '../images/NASADYA_Khadeeja-Malik_Headshot_1.jpg'
+import Dan from '../images/NASADYA_Dan Duffy_Headshot.jpg'
+import Nate from '../images/Nate-Headshot_1.jpg'
+import Madhu from '../images/NASADYA_MadhuDevi.jpg'
+
+
+const Logo = s.img`
+    display: block;
+    margin: auto;
+    width: 30vw;
+    padding-top: 2vw;
+    padding-bottom: 2vw;
+`
+
+const Headshot = s.img`
+    object-fit: cover;
+    width: 12vw;
+    height: 12vw;
+`
+
+const Columns = s.div`
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    margin: 2vw auto;
+`
+
+const ColumnText = s.div`
+    flex: 50vw;
+    width: 50%;
+`
+
+const Name = s.figcaption`
+    font-family: roboto;
+    font-weight: bold;
+    color: ${GREEN};
+    font-size: 20px;
+`
```

Comment:
This is actually great because figcaption will be annotated by accessibility browsers : D

👍 👍 👍 👍 👍